browser/src/control/Control.AboutDialog.ts

```diff
@@ -129,6 +106,74 @@ class AboutDialog {
 				productNameWithURL,
 			);
 
+		// COOLWSD version
+		const coolwsdEl = content.querySelector('#coolwsd-version');
+		if (coolwsdEl && info.coolwsdVersion) {
+			coolwsdEl.textContent = info.coolwsdVersion;
+			if (info.coolwsdHash) {
+				const span = document.createElement('span');
+				const a = document.createElement('a');
+				a.href = `https://github.com/CollaboraOnline/online/commits/${info.coolwsdHash}`;
+				a.target = '_blank';
+				a.textContent = info.coolwsdHash;
+				span.appendChild(document.createTextNode(' git hash:\xA0'));
+				span.appendChild(a);
+				coolwsdEl.appendChild(span);
+			}
+		}
+
+		const lokitExtra = content.querySelector('#lokit-extra');
+		if (lokitExtra) {
+			const span = document.createElement('span');
+			span.textContent = _('built on\xA0');
+			const a = document.createElement('a');
+			a.href = 'https://col.la/lot';
+			a.target = '_blank';
+			a.textContent = _('a great technology base');
+			lokitExtra.appendChild(span);
+			lokitExtra.appendChild(a);
+		}
+
+		// LOKit version
+		const lokitEl = content.querySelector('#lokit-version');
+		if (lokitEl && info.lokitVersion) {
+			lokitEl.textContent = info.lokitVersion;
+			if (info.lokitHash) {
+				const span = document.createElement('span');
+				const a = document.createElement('a');
+				a.href = `https://git.libreoffice.org/core/+log/${info.lokitHash}`;
+				a.target = '_blank';
+				a.textContent = info.lokitHash.substring(0, 10);
+				span.appendChild(document.createTextNode(' git hash:\xA0'));
+				span.appendChild(a);
+				lokitEl.appendChild(span);
+			}
+		}
+
+		// Served By
+		const servedBy = content.querySelector('#served-by');
+		if (servedBy && (info.osInfo || info.serverId)) {
+			servedBy.innerHTML = `
```

Comment:
We should avoid `innerHTML` with any non-static content (even if it comes from our own server).
Please build using JS.